exercises/practice/twelve-days/.docs/hints.md

```diff
@@ -0,0 +1,16 @@
+# Hints
+
+# General
+
+Have you tried to break your logic up into one or more helper functions?
```

Comment:
```suggestion
Have you tried to break your logic up into one or more helper functions?
This type of song is known as a [cumulative song][cumulative song], so it could be particularly well suited to being "broken up".
```